app/controllers/users_controller.rb

```diff
@@ -1,37 +1,78 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: [:new, :create]
```

Comment:
```suggestion
  before_action :logged_in_user, except: %i(new create)
```
Chuyển sang dùng như này nha